### PR TITLE
Supported custom I2C + minor changes

### PR DESCRIPTION
**Pull request by FStefanni**
Hi,

this PR proposes the following improvements:

1. Added support to use a custom I2C instance. Default is to use the Wire instance.
2. Added header to support installation with arduino-cli
3. Added explicit constant for SHT85

They are all backward compatible improvements.

Regards.

### CODE REVIEW

File: SHTSensor.h [hunk 1]
```diff
@@ -241,10 +245,12 @@ class SHTI2cSensor : public SHTSensorDriver {
   float mY;
   float mZ;
   uint8_t mCmd_Size;
+  TwoWire & _wire;
```
Comment (winkj) [inline SHTSensor.h:248]: we're using 'm' prefixes for all member variables, so for _wire should be mWire

**Comment by winkj** [comment]
Looks good to me, thanks a lot for the contribution. I added one comment - the variable name ```_wire``` should be ```mWire``` for consistency reasons.

Other than that, since these are three independent changes, it should be three independent commits, each implementing one of the changes.

Let me know if you can to prepare an updated PR addressing those two things, otherwise I can merge it manually

**Comment by FStefanni** [comment]
Hi,

done the suggested improvements.

Regards.

**Comment by winkj** [comment]
Thanks for the update; I'll pull in 81ebeb1 and 12acf60.

I'll rewrite 1dffe18 to be consistent with other sensors (individual constant, and then same break in autodetect); also, it looks like you introduced some extra change in there to rename mWire back to _wire

**Comment by winkj** [comment]
I've merged 81ebeb1 and 12acf60, and submitted a PR for SHT85 support (#21).

Thanks a lot for your contribution!

**Comment by FStefanni** [comment]
Hi,

perfect. Thank you for accepting the changes.

Regards.

**Pull request closed without merging**